src/_pytest/hookspec.py

```diff
@@ -472,7 +472,7 @@ def pytest_unconfigure(config):
 # -------------------------------------------------------------------------
 
 
-def pytest_assertrepr_compare(config, op, left, right):
+def pytest_assertrepr_compare(config, op, left, right, expl):
```

Comment:
Please update the docstring with `expl` as well. 😁

Reply:
Would need more updating in general then, especially for when we go with the return-string-to-skip special postprocessing approach.